### ISSUE TITLE
Stray `{ref, ok}` message can kill the writer process

### ISSUE THREAD

**Issue by lukebakken** [rabbitmq/rabbitmq-server#9991]
### Describe the bug

Complete details can be found in this discussion:

https://github.com/rabbitmq/rabbitmq-server/discussions/9803

### Reproduction steps

No reproduction steps exist at this time.

### Expected behavior

The source of the stray message is found, or the message is logged and then ignored.

### Additional context

@gomoripeti says that there is code in the `ra` library that fits the bill for the format of the stray message ([link](https://github.com/rabbitmq/rabbitmq-server/discussions/9803#discussioncomment-7653240)). Waiting on him to point out what code it is. This may, of course, be a message from within OTP itself.

**Comment by lukebakken** [comment]
This feels like a `warning` level sort of message, maybe?

**Comment by michaelklishin** [comment]
@lukebakken logging a warning works for me https://github.com/rabbitmq/rabbitmq-server/pull/9994.

Those who have a way to reproduce and are interested in digging in now have a place to add more logging and tracing.

**Comment by lukebakken** [comment]
I'm going to re-open this issue to devote some time (at some point) to figure out the root cause.

@michaelklishin will having this issue open affect release notes? I can open a new issue if so.

**Comment by michaelklishin** [comment]
Don't worry about the release notes, we can leave it open

**Comment by kjnilsson** [comment]
I can't see any usages in Ra of `{ref, ok}` the closest I have is https://github.com/rabbitmq/ra/blob/ba3293a3ea29e6aa738726ba22e951f1ce8e7af4/src/ra_server_sup_sup.erl#L243
but that should be a proper `reference()` not the atom `ref`.

**Comment by lukebakken** [comment]
@kjnilsson yes, the error in the logs shows an actual reference, not the atom `ref`

**Comment by michaelklishin** [comment]
I've closed this because as of #9991, we hope that the writer process does not die. If someone wants to investigate where these messages originate from, you are welcome to do it.

It's just that this specific issue, as it is worded right now, should be addressed.
